src/cli/index.js

```diff
@@ -29,7 +29,7 @@ const args = process.argv.slice(2, doubleDashIndex === -1 ? process.argv.length
 const endArgs = doubleDashIndex === -1 ? [] : process.argv.slice(doubleDashIndex + 1, process.argv.length);
 
 // set global options
-commander.version(version);
+commander.version(version, '-v, -V, --version');
```

Comment:
let's not alias -v/-V to --version yet because we have not agreed whether people will confuse it with --verbose.